core/domain/exp_jobs_one_off_test.py

```diff
@@ -1693,3 +1693,255 @@ def test_validation_job_fails_for_invalid_schema_version(self):
+
+        with self.swap(state_domain.SubtitledHtml, 'validate', mock_validate):
+            state1.update_content(
+                state_domain.SubtitledHtml.from_dict(content1_dict))
+            state2.update_content(
+                state_domain.SubtitledHtml.from_dict(content2_dict))
+            state2.update_interaction_id('DragAndDropSortInput')
+            state2.update_interaction_customization_args(
+                customization_args_dict)
+            state2.update_interaction_answer_groups([answer_group_dict])
+            state2.update_written_translations(
+                state_domain.WrittenTranslations.from_dict(
+                    written_translations_dict))
+
+            exp_services.save_new_exploration(self.albert_id, exploration)
+
+            job_id = (
+                exp_jobs_one_off
+                .ExplorationMathTagValidationOneOffJob.create_new())
+            exp_jobs_one_off.ExplorationMathTagValidationOneOffJob.enqueue(
+                job_id)
+            self.process_and_flush_pending_tasks()
+
+        actual_output = (
+            exp_jobs_one_off
+            .ExplorationMathTagValidationOneOffJob.get_output(job_id))
+
+        actual_output_list = ast.literal_eval(actual_output[0])
+        no_of_invalid_tags = 0
+        for output in actual_output_list[1]:
+            list_starting_index = output.find('[')
+            list_finishing_index = output.find(']')
+            stringified_error_list = (
+                output[list_starting_index:list_finishing_index + 1])
+            length_of_list = len(stringified_error_list.split('>,'))
+            no_of_invalid_tags = (
+                no_of_invalid_tags + length_of_list)
+        self.assertEqual(no_of_invalid_tags, 12)
+
+
+    def test_no_action_is_performed_for_deleted_exploration(self):
+        """Test that no action is performed on deleted explorations."""
+
+        exploration = exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title=self.EXP_TITLE, category='category')
+
+        exploration.add_states(['State1'])
+        invalid_html_content = (
+            '<p>Value</p><oppia-noninteractive-math></oppia-noninteractive-m'
+            'ath>')
+        content_dict = {
+            'html': invalid_html_content,
+            'content_id': 'content'
+        }
+        state1 = exploration.states['State1']
+
+        with self.swap(state_domain.SubtitledHtml, 'validate', mock_validate):
+            state1.update_content(
+                state_domain.SubtitledHtml.from_dict(content_dict))
+            exp_services.save_new_exploration(self.albert_id, exploration)
+
+        exp_services.delete_exploration(self.albert_id, self.VALID_EXP_ID)
+
+        run_job_for_deleted_exp(
+            self,
+            exp_jobs_one_off.ExplorationMathTagValidationOneOffJob)
+
+    def test_explorations_with_valid_math_tags(self):
+        """Find invalid math tags in exploration with no invalid math tags."""
```

Comment:
Test docstrings should start with "Test that". Please fix, here and (possibly) elsewhere.

Reply:
done